### PR TITLE
Fix config Log examples

### PR DESCRIPTION
**Pull request by Kiina**

### CODE REVIEW

**Comment by roramirez** [comment]
Please, resolve the lint problems. I suggest update your branch.

**Comment by Kiina** [comment]
Well then the lint errors are on master? Because my changes don't change any of the files that fail for the linter

File: modules/README.md [hunk 1]
```diff
@@ -681,7 +681,17 @@ The Magic Mirror contains a convenience wrapper for logging. Currently, this log
...
 ````javascript
-Log.info('info');
-Log.log('log');
-Log.error('error');
+Log.info('This is an Information from the Module');
+Log.log('This is a normal Log from the Module');
+Log.error('This is an Error from the Module');
 ````
+
+You will find these logs in your Electron console.
```
Comment (roramirez) [inline modules/README.md]: Nice... but if you using the serveronly can you see in your web browser inspector too ;)

**Comment by roramirez** [comment]
Yes @Kiina, you need merge the develop upstream branch inside your Pull Request branch.

**Comment by MichMich** [comment]
I had to revert this PR since I found out this was going to the master branch afterwards. Please only send your PR's to the develop branch. Thanks!

**Pull request merged**